<launch>
    <node pkg="global_planning" type="static_cycle_bus.py" name="static_cycle_bus_plan" output="screen">
        <param name="cyclebus_pathfile" value="/home/sin/Data/datas/maps/20190701xuancheng/trajs/" />
    </node>
</launch>
